.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,9 @@
-### Your checklist for this pull request
+<!--### Your checklist for this pull request
```

Comment:
I guess with this it wont even be formatted nicely? Since the comment is on the formatting character.

Reply:
Just checked it. I think it's fine 😄